contracts/truefi2/TrueFiPool2.sol

```diff
@@ -274,21 +279,18 @@ contract TrueFiPool2 is ITrueFiPool2, IPauseableContract, ERC20, UpgradeableClai
     /**
      * @dev only FixedTermLoanAgency, or CreditAgency can perform borrowing or repaying
```

Comment:
Let's make this comment more robust to future agencies we might add. We should also clarify that this modifier is only used to gatekeep borrowing, not repaying.
```suggestion
     * @dev only agencies allowed by the borrowingMutex (like FixedTermLoanAgency) can borrow
```